NEWS.md

```diff
@@ -120,6 +120,8 @@
 
 15. `rbind` and `rbindlist` of zero-row items now retain (again) the unused levels of any (zero-length) factor columns, [#3508](https://github.com/Rdatatable/data.table/issues/3508). This was a regression in v1.12.2 just for zero-row items. Unused factor levels were already retained for items having `nrow>=1`. Thanks to Gregory Demin for reporting.
 
+16. Running the `:=` examples with `example` and `local=TRUE` works as expected, [#2972](https://github.com/Rdatatable/data.table/issues/2972). Thanks @vlulla for the report.
```

Comment:
'as expected' doesn't mean much to a reader of news. 'works rather than error'.  The item can be shortened by just writing `example(`:=`, local=TRUE)` at the beginning, because 'with' isn't quite as clear.